experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -33,6 +33,18 @@ import {
 } from "src/types/globalTypes";
 
 describe("FormAudience", () => {
+  const origWindowOpen = global.window.open;
+  let mockWindowOpen: any;
+
+  beforeEach(() => {
+    mockWindowOpen = jest.fn();
+    global.window.open = mockWindowOpen;
+  });
+
+  afterEach(() => {
+    global.window.open = origWindowOpen;
+  });
+
   it("renders without error", async () => {
```

Comment:
oh why these changes? 🤔

Reply:
I'm using `window.open` so that it opens a new tab with [whattrainisit](whattrainisitnow.com/), so this sets up a mock for window.open :D